tests/Functional/ContentRenderTestBase.php

```diff
@@ -202,7 +202,7 @@ protected function createContactEntity(string $name, string $bundle, int $status
       ],
       'status' => $status,
     ]);
-
+    $contact->save();
```

Comment:
Doesn't this work also without saving the entity in here? I wouldn't maybe change it in here, without checking everywhere else if that's ok and if similar methods should be changed too along aside it. If we need to save the entity this method creates, we can do it in the caller method, as the entity is returned.

Reply:
Our create* methods work inconsistent - createMedia*() save media and return saved entity

Reply:
I decided to keep similar methods consistent and add the contact->save in the end. The only other place where this is used is on the news test and so it doesn't really make any difference in that regard